### DISCUSSION TITLE
I can't get into "localhost:18083"

### DISCUSSION THREAD

**Discussion by yangruomao** [emqx/emqx#5097] [Q&A]
### Use process
After `emqx start`, No errors,And i use `emqx console` cmd shows emqx is started!;
if i 'emqx stop', cmd shows `Node 'emqx@127.0.0.1' not responding to pings.`
After these operator, file /log/emqx.log.1 is NULL;

### Picture
![image](https://user-images.githubusercontent.com/72311031/121983967-bfcb7c00-cdc4-11eb-8887-a4e2edf57a25.png)
![image](https://user-images.githubusercontent.com/72311031/121983979-c4903000-cdc4-11eb-97ea-ef4963d34537.png)
![image](https://user-images.githubusercontent.com/72311031/121984006-d07bf200-cdc4-11eb-8e34-18c7df7e0962.png)
![image](https://user-images.githubusercontent.com/72311031/121984018-d540a600-cdc4-11eb-905d-fe58372389c7.png)

**Comment by k32** [comment]
Hello,
Could you start emqx using `emqx console` command and post the output?

**Reply by ghj-lsz** [reply]
![image](https://user-images.githubusercontent.com/87603890/141879667-5acc60de-005d-429a-9929-efbbf6909875.png)
then the Erlang window will close automatically

**Comment by 45677491** [comment]
Have you solved the problem?

**Comment by zmstone** [comment]
4.4
<img width="994" alt="image" src="https://user-images.githubusercontent.com/164324/155295692-da59c9ce-72b1-4b6c-8ec7-636eb88fca3b.png">

5.0
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/164324/155292113-d06e49c2-9586-41f1-a7b7-f70608130d3b.png">

**Reply by mikite** [reply]
This issue still happens with **emqx-4.4.1-otp23.0-windows-amd64** on Windows10 profession .
